commit 851dd4ddc5aeb1ee517145d9e3334c2017595321
Author: Harald Hoyer <harald@redhat.com>
Date:   Fri Nov 12 14:39:14 2010 +0100

    udev-rules.c: change import property buffer to 16384 bytes
    
    import_program_into_properties() should have the same line length as
    import_file_into_properties()
    
    see also https://bugzilla.redhat.com/show_bug.cgi?id=652318

diff --git a/udev/udev-rules.c b/udev/udev-rules.c
index 52f121c..ab7b6ae 100644
--- a/udev/udev-rules.c
+++ b/udev/udev-rules.c
@@ -751,7 +751,7 @@ static int import_program_into_properties(struct udev_device *dev, const char *p
 {
 	struct udev *udev = udev_device_get_udev(dev);
 	char **envp;
-	char result[4096];
+	char result[UTIL_LINE_SIZE];
 	size_t reslen;
 	char *line;
 
